Subject: link libspice server with libm libpthread
From: Michael Tokarev <mjt@tls.msk.ru>
Forwarded: yes

Upstream makefile apparently forgot to link libspice-server
with -lm -lpthread -- it uses symbols from these libraries.
These libs are detected by configure and stored in
$(SPICE_NONPKGCONFIG_LIBS) make variable, but this variable
is never referenced.  Add it to server/Makefile.am.


diff --git a/server/Makefile.am b/server/Makefile.am
index 1284704..b17b545 100644
--- a/server/Makefile.am
+++ b/server/Makefile.am
@@ -48,6 +48,7 @@ libspice_server_la_LIBADD =				\
 	$(SLIRP_LIBS)					\
 	$(SSL_LIBS)					\
 	$(Z_LIBS)					\
+	$(SPICE_NONPKGCONFIG_LIBS)			\
 	$(NULL)
 
 libspice_server_la_SOURCES =			\
